src/Service.Tests/TestHelper.cs

```diff
@@ -240,26 +240,39 @@ public static void AddMissingEntitiesToConfig(RuntimeConfig config, string entit
 
         /// <summary>
         /// Utility method that reads the config file for a given database type and constructs a
-        /// new config file with changes just in the host mode section.
+        /// new config file with custom changes as specified in the method parameters.
         /// </summary>
         /// <param name="configFileName">Name of the new config file to be constructed</param>
         /// <param name="hostModeType">HostMode for the engine</param>
         /// <param name="databaseType">Database type</param>
-        public static void ConstructNewConfigWithSpecifiedHostMode(string configFileName, HostModeType hostModeType, string databaseType)
+        public static void ConstructNewConfigWithCustomSettings(
```

Comment:
Just noting that #1402 will require a revisit of this file due to the structure changes of the config types.